plugins/processors/ifname/ifname.go

```diff
@@ -0,0 +1,370 @@
+package ifname
+
+import (
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/snmp"
+	si "github.com/influxdata/telegraf/plugins/inputs/snmp"
+	"github.com/influxdata/telegraf/plugins/processors"
+	"github.com/influxdata/telegraf/plugins/processors/reverse_dns/parallel"
+)
+
+var sampleConfig = `
+  ## Name of tag holding the interface number
+  # tag = "ifIndex"
+
+  ## Name of output tag where service name will be added
+  # dest = "ifName"
+
+  ## Name of tag of the SNMP agent to request the interface name from
+  # agent = "agent"
```

Comment:
Can't this be default set to `agent_host` as this is the tag that is being returned from `snmp` plugin? It would reduce the need to set this value every time.

Reply:
I don't think we should change the default now that this has been released.  Changing the default would silently break anyone who is using the default.

Reply:
I can't figure out why anyone would have "agent" as default? There is no plugin that provides this tag by default.

Reply:
You have a valid point that it's unlikely anyone is using the default.  They would have to rename the tag from the snmp input or the sflow input (which uses "agent_address").